### PR TITLE
Nl webren

### PR DESCRIPTION
**Pull request by Nathan-Ladwig**
The two commits that I would like to merge are corrections for the A_Z databases and Subjects files and page tests 8 and 9 from the spreadsheet.

### CODE REVIEW

**Comment by h-parekh** [comment]
@Nathan-Ladwig  It appears like the first five commits in this PR have already been merged into master from a previous  PR https://github.com/ndlib/QA_tests/pull/157

File: spec/webRenovation/functional/func_webRenovation_spec.rb [hunk 1]
```diff
@@ -185,3 +185,21 @@
     expect(coursesPage).to be_on_page
   end
 end
+
+
+feature 'User Navigation', js: true do
+  scenario 'Header Tab Navigation', :read_only, :smoke_test do
+    page.driver.browser.js_errors = false
```
Comment (h-parekh) [inline spec/webRenovation/functional/func_webRenovation_spec.rb]: Let's remove this error suppression, trying to make the QA test tighter.

File: spec/webRenovation/functional/func_webRenovation_spec.rb [hunk 2]
```diff
@@ -185,3 +185,21 @@
...
+    visit '/'
+    header_checks = WebRenovation::Pages::HeaderChecks.new
+    expect(header_checks).to be_on_page
+  end
+  scenario 'All Feature in Tab', :read_only, :smoke_test do
+    page.driver.browser.js_errors = false
```
Comment (h-parekh) [inline spec/webRenovation/functional/func_webRenovation_spec.rb]: Remove the error suppression.

File: spec/webRenovation/pages/base_page.rb [hunk 3]
```diff
@@ -23,21 +23,48 @@ def valid_header?
...
-          if @loggedin
-            page.has_link?('My Account')
-          else
-            page.has_link?('Login')
-          end
-          find('a', id: 'header-search-button').visible?
-          find_link('Hours', href: '/hours').visible?
         end
+        within('.uNavigation') do
+          find_by_id('research').trigger('click')
```
Comment (h-parekh) [inline spec/webRenovation/pages/base_page.rb]: Are you clicking the 'research' button to make the sub-buttons visible?

Comment (Nathan-Ladwig) [inline spec/webRenovation/pages/base_page.rb]: I'm clicking the buttons to show the sub-menus in order to check their headers, yes.

**Pull request merged**